movement/movement_config.h

```diff
@@ -29,12 +29,8 @@
 
 const watch_face_t watch_faces[] = {
     simple_clock_face,
-    world_clock_face,
```

Comment:
I would revert this file to its main branch state; if this gets merged as-is it'll overwrite the default faces.

Reply:
good point! done.